test/SlowTests/Tests/Linq/CanQueryAndIncludeRevisions.cs

```diff
@@ -0,0 +1,1982 @@
+                }
+                
+                using (var session = store.OpenAsyncSession())
+                {
+                    var query = await session.Advanced
+                        .AsyncRawQuery<User>("from Users as u include revisions($p0, $p1, $p2)")
+                        .AddParameter("p0","u.FirstRevision")
+                        .AddParameter("p1","u.SecondRevision")
+                        .AddParameter("p2","u.ThirdRevision")
+                        .ToListAsync();
+                    
+                    var revision1 = await session.Advanced.Revisions.GetAsync<User>(cvList[0]);
+                    var revision2 = await session.Advanced.Revisions.GetAsync<User>(cvList[1]);
+                    var revision3 = await session.Advanced.Revisions.GetAsync<User>(cvList[2]);
+                
+                    Assert.NotNull(revision1);
+                    Assert.NotNull(revision2);
+                    Assert.NotNull(revision3);
+                    
+                    Assert.Equal(1, session.Advanced.NumberOfRequests);
+                }
+            }
+        }
+
+        [Fact]
+        public void Query_RawQueryWithParameters_AliasSyntaxError()
+        {
+            using (var store = GetDocumentStore())
+            {
+                var cvList = new List<string>();
+
+                const string id = "users/omer";
+
+                RevisionsHelper.SetupRevisions(Server.ServerStore, store.Database).Wait();
+
+                using (var session = store.OpenSession())
+                {
+                    session.Store(new User
+                        {
+                            Name = "Omer",
+                        },
+                        id);
+                    
+                    session.SaveChanges();
+                }
+
+                using (var session = store.OpenSession())
+                {
+                    var error =  Assert.ThrowsAny<RavenException>(  () => session.Advanced
+                        .RawQuery<User>("from Users as u include revisions(u.FirstRevision, x.SecondRevision)")
+                        .ToList());
+                        
+                    Assert.Contains("System.InvalidOperationException: Cannot include revisions for related parameter 'x.SecondRevision', Parent alias is different than include alias 'u' compare to 'x'."
+                        , error.Message);
+                }
+                
+                using (var session = store.OpenSession())
+                {
+                    
+                    var error =  Assert.ThrowsAny<RavenException>( () =>  session.Advanced
+                        .RawQuery<User>("from Users as u include revisions($p0, $p1, $p2)")
+                        .AddParameter("p0", "u.FirstRevision")
+                        .AddParameter("p1", "u.SecondRevision")
+                        .AddParameter("p2", "x.ThirdRevision")
```

Comment:
@ppekrol How do you read this? I would lean toward just killing passing this via parameters entirely, given the alias situation

Reply:
I do not recall us having similar syntax elsewhere. Maybe it just works out of the box? @omerlewitz you had to add support for this explicitly? If yes, probably we should drop it. Or maybe there is some case where this would be useful?